### PR TITLE
Use .NET Core 2.1 & minor fixes.

### PR DESCRIPTION
**Pull request by RheaAyase**
* Switch to netcore 2.1
* Fixed basic naming
* Fixed username regex (and its usage)

_(Please use rebase merge)_

### CODE REVIEW

File: src/BotTerminator.cs [hunk 1]
```diff
@@ -149,16 +149,16 @@ public async Task StartNewBanUpdateLoopAsync()
...
 						if (p.LinkFlairText != "Banned") return;
 						if (p.IsHidden) return;
 						posts.Add(p);
 					});
-					foreach (Post p in posts)
+					foreach (Post post in posts)
 					{
 
-						Match m = re.Match(p["url"].Value<String>().Trim());
+						Match m = UserUrlRegex.Match(post["url"].Value<String>().Trim());
```
Comment (RheaAyase) [inline src/BotTerminator.cs]: This line actually doesn't compile, you can't use `Post["url"]` like that.

Comment (RheaAyase) [reply src/BotTerminator.cs]: Fixed.

**Comment by RheaAyase** [comment]
Well now that you went around and changed half of the code I have to fix all of this because it can no longer be merged. Next time please deal with pull requests first before rewriting half of the code.

**Comment by justcool393** [comment]
I'm so sorry about that. I didn't mean to waste your time.

So because most of the diff is now conflicting, I think the only thing that needs to be done here is move to .NET Core 2.1.

**Pull request merged**